__tests__/__integration__/imperative.integration.test.ts

```diff
@@ -0,0 +1,226 @@
+        let TEST_ENVIRONMENT: ITestEnvironment;
+        let SYSTEM_PROPS: ITestPropertiesSchema;
+
+        beforeAll(async () => {
+            TEST_ENVIRONMENT = await TestEnvironment.setUp({
+                testName: "zosmf_create_profile"
+            });
+
+            SYSTEM_PROPS = TEST_ENVIRONMENT.systemTestProperties;
+        });
+
+        afterAll(async () => {
+            await TestEnvironment.cleanUp(TEST_ENVIRONMENT);
+        });
+
+        afterEach(async () => {
+            const opts = ["CreateProfileSystemTest"];
+            try {
+                runCliScript(__dirname + "/__scripts__/imperative_zosmf_delete_profile.sh", TEST_ENVIRONMENT, opts);
+            // tslint:disable-next-line: no-empty
+            } catch (err) { }
+        });
+
+        it("should successfully create a profile", async () => {
+            const opts = [
+                "CreateProfileSystemTest",
+                "--host", SYSTEM_PROPS.zosmf.host,
+                "--port", SYSTEM_PROPS.zosmf.port,
+                "--user", SYSTEM_PROPS.zosmf.user,
+                "--password", SYSTEM_PROPS.zosmf.pass,
+                "--reject-unauthorized", SYSTEM_PROPS.zosmf.rejectUnauthorized
+            ];
+
+            const response = runCliScript(__dirname + "/__scripts__/imperative_zosmf_create_profile.sh",
+                TEST_ENVIRONMENT, opts
+            );
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+            expect(response.stdout.toString()).toContain("Profile created successfully!");
+            expect(response.stdout.toString()).toContain(SYSTEM_PROPS.zosmf.host);
+            expect(response.stdout.toString()).toContain(SYSTEM_PROPS.zosmf.port);
+            expect(response.stdout.toString()).toContain(SYSTEM_PROPS.zosmf.user);
+            expect(response.stdout.toString()).toContain(SYSTEM_PROPS.zosmf.pass);
+        });
+
+        it("should successfully create a profile without username, password, or host", async () => {
+            const opts = [
+                "CreateProfileSystemTest",
+                "--port", SYSTEM_PROPS.zosmf.port,
+                "--reject-unauthorized", SYSTEM_PROPS.zosmf.rejectUnauthorized
+            ];
+
+            const response = runCliScript(__dirname + "/__scripts__/imperative_zosmf_create_profile.sh",
+                TEST_ENVIRONMENT, opts
+            );
+            expect(response.stderr.toString()).toBe("");
+            expect(response.status).toBe(0);
+            expect(response.stdout.toString()).toContain("Profile created successfully!");
+            expect(response.stdout.toString()).toContain(SYSTEM_PROPS.zosmf.port);
+        });
+    });
+
+    describe("create ssh profile", () => {
+
+        // Create a separate test environment for no profiles
+        let TEST_ENVIRONMENT: ITestEnvironment;
+        let SYSTEM_PROPS: ITestPropertiesSchema;
+
+        beforeAll(async () => {
```

Comment:
Same comment about `beforeAll` and `afterAll` as above

Reply:
Same answer as above.